autospec/build.py

```diff
@@ -164,6 +165,26 @@ def parse_build_results(filename, returncode, filemanager):
             success = 1
 
 
+def get_uniqueext(dirn, dist, name):
+    """
+    Find a unique name to create mock chroot without reusing an old one
+    """
+    # Default to tarball name
+    resultsdir = os.path.join(dirn, "{}-{}".format(dist, name))
```

Comment:
Thoughts on wrapping os.mkdir in a try and checking for FileExistsError instead? Prevents a race condition I think when it is run at the same time.

Reply:
Ah, true. Ok.

Reply:
Hrm well the usual operation is leaving the directory behind... Could make the test a little function that would switch between exists and mkdir depending on if --no-cleanup-after is being used? Starting to feel more okay with the race =P.

Reply:
the problem with the os.mkdir approach is that you need to be root to touch /usr/bin/mock/